packages/design-tokens/src/css/typography.css

```diff
@@ -57,15 +57,6 @@
   --typography-s-heading-sm-line-height: var(--line-height-4);
   --typography-s-heading-sm-font-weight: var(--font-weight-bold);
   --typography-s-heading-sm-letter-spacing: var(--letter-spacing-0);
-
-  /**
-   * @deprecated [#1] since version 1.9 [#2].
-   * [#3] Will be deleted in version 2.0.
-   */
-  --typography-s-heading-sm-regular-font-size: var(--font-size-4);
-  --typography-s-heading-sm-regular-line-height: var(--line-height-4);
-  --typography-s-heading-sm-regular-font-weight: var(--font-weight-regular);
-  --typography-s-heading-sm-regular-letter-spacing: var(--letter-spacing-0);
```

Comment:
Removing from CSS stylesheet. Ensured they are not used in the codebase

<img width="510" alt="Screenshot 2025-05-25 at 2 19 28 PM" src="https://github.com/user-attachments/assets/b2802e25-848a-44d8-a6bd-af6da2ffac5b" />